README.md

```diff
@@ -258,80 +259,82 @@ and disables the auto-detection feature:
-`<lintStrict>` | boolean | helm.lint.strict | false | run lint command with strict option (fail on lint warnings)
-`<lintQuiet>` | boolean | helm.lint.quiet | false | run lint command with quiet option (print only warnings and errors)
+`<helmExtraRepos>` | list of [HelmRepository](./src/main/java/io/kokuwa/maven/helm/pojo/HelmRepository.java) |                    | false | adds extra repositories while init
+`<uploadRepoStable>`| [HelmRepository](./src/main/java/io/kokuwa/maven/helm/pojo/HelmRepository.java) |                    | false | Upload repository for stable charts
+`<uploadRepoSnapshot>`| [HelmRepository](./src/main/java/io/kokuwa/maven/helm/pojo/HelmRepository.java) |                    | false | Upload repository for snapshot charts (determined by version postfix 'SNAPSHOT')
+`<lintStrict>` | boolean | helm.lint.strict   | false | run lint command with strict option (fail on lint warnings)
+`<lintQuiet>` | boolean | helm.lint.quiet    | false | run lint command with quiet option (print only warnings and errors)
 `<addDefaultRepo>` | boolean | helm.init.add-default-repo | true | If true, stable repo (<https://charts.helm.sh/stable>) will be added
 `<addUploadRepos>` | boolean | helm.init.add-upload-repos | false | If true, upload repos (uploadRepoStable, uploadRepoSnapshot) will be added, if configured
-`<skip>` | boolean | helm.skip | false | skip plugin execution
-`<skipInit>` | boolean | helm.init.skip | false | skip init goal
-`<skipLint>` | boolean | helm.lint.skip | false | skip lint goal
+`<skip>` | boolean | helm.skip          | false | skip plugin execution
+`<skipInit>` | boolean | helm.init.skip     | false | skip init goal
+`<skipLint>` | boolean | helm.lint.skip     | false | skip lint goal
 `<skipTemplate>` | boolean | helm.template.skip | false | skip template goal. Default value is true due to the dry-run goal
-`<skipDryRun>` | boolean | helm.dry-run.skip | false | skip dry-run goal
+`<skipDryRun>` | boolean | helm.dry-run.skip  | false | skip dry-run goal
 `<skipDependencyBuild>` | boolean | helm.dependency-build.skip | false | skip dependency-build goal
 `<skipDependencyUpdate>` | boolean | helm.dependency-update.skip | false | skip dependency-update goal
-`<skipPackage>` | boolean | helm.package.skip | false | skip package goal
-`<skipUpload>` | boolean | helm.upload.skip | false | skip upload goal
+`<skipPackage>` | boolean | helm.package.skip  | false | skip package goal
+`<skipUpload>` | boolean | helm.upload.skip   | false | skip upload goal
 `<insecure>` | boolean | helm.upload.insecure | false | Skip tls certificate checks for the chart upload.
-`<skipInstall>` | boolean | helm.install.skip | false | skip install goal
+`<uploadVerification>` | boolean | helm.upload.verification | false | verify if chart was uploaded to the remote repository
+`<uploadVerificationTimeout>` | Integer | helm.upload.timeout | false | set the timeout limit (in seconds) for verification to be attempted
```

Comment:
this is the only real change to this file, the rest are weird spacing changes by IntelliJ

Reply:
You might want to fix these spacing changes prior to the merge to the forked repo

Reply:
fixed!